lib/flow.js

```diff
@@ -165,21 +165,22 @@ const pdata = function pdata(cb) {
   cb(null, this);
 };
 
+const setupReporterOptions = (self, instance) => {
```

Comment:
The arrow function requires the passing of the context `self` parameter. If it's a classical `function` it could be called using `call` or `apply`. Since the approach in this file is currently to bind functions to the context, it is more consistent to do that here as well.